Conceptpower+Spring/src/main/java/edu/asu/conceptpower/rest/ConceptSearch.java

```diff
@@ -148,6 +149,43 @@ private void initBinder(WebDataBinder binder) {
                 HttpStatus.OK);
     }
 
+    /**
+     * 
+     * This method is used for setting the correct values for concept search
+     * parameters.
+     * 
+     * @param conceptSearchParameters
+     * @param page
+     * @return
+     * @throws IntrospectionException
+     * @throws IllegalAccessException
+     * @throws IllegalArgumentException
+     * @throws InvocationTargetException
+     * @throws NumberFormatException
+     */
+    private void preprocessConceptSearchParameterBean(ConceptSearchParameters conceptSearchParameters)
```

Comment:
Passed conceptSearchParameter object as a parameter to load the typeId based on the type uri.